pinot-core/src/main/java/org/apache/pinot/core/query/request/ServerQueryRequest.java

```diff
@@ -77,12 +85,16 @@ public ServerQueryRequest(InstanceRequest instanceRequest, ServerMetrics serverM
     _timerContext = new TimerContext(_queryContext.getTableName(), serverMetrics, queryArrivalTimeMs);
   }
 
+  /**
+   * This is called from the grpc server to create a ServerQueryRequest from the grpc request.
```

Comment:
Might be worth clarifying that this is the SSE gRPC server

Reply:
The constructor above indicates in its javadoc that it is the one used by MSE. Anyway I'm adding more javadoc to make it easier to understand.